src/main/java/mate/academy/service/ProcessDataService.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.service;
+
+import java.util.List;
+import mate.academy.model.FruitTransaction;
+import mate.academy.strategy.TransactionStrategy;
+
+public interface ProcessDataService {
```

Comment:
<img width="508" alt="image" src="https://github.com/mate-academy/jv-fruit-shop/assets/72043323/a2a25e65-867c-4fd9-a8c7-902a304a7c6c">
It's not mandatory, but you can refactor code to simplify interface naming.